### PR TITLE
Made some grammatical and spelling fixes in the readme.md file

### PR DESCRIPTION
**Pull request by mkjha482**
Grammatical and spelling updates in the readme file.

Screenshot-updated-readme-file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
  - Easy Level;
  - Medium Level;
  - Hard Level;
  - Refactoring Level.
 
+Look for all levels of exercises:
+
 To contribute to this project, create a PR. If tested and approved, your code will be inserted!
 
 *This project is created for Hacktober Fest 2020. [Read more](https://hacktoberfest.digitalocean.com/)*
```
Comment (SamuelBFavarin) [inline README.md]: What do you think to update the Hacktober Fest year, and also the event logo? 😄

**Review by SamuelBFavarin — CHANGES_REQUESTED** [review]
Thanks for helping me keeping with this repo! I'll appreciate it if you contribute at least exercise too.

File: README.md (hunk 1 above)
Comment (echoMayank) [reply README.md]: sure, I apologize for failing to notice the year and logo. I will correct it as well. I will also try to add an exercise as well with a different PR. I'll leave this one for the documentation.

**Comment by echoMayank** [comment]
I have made the requested changes in the documentation as requested. Pending for review and merge.
Also, For the exercise, should I add a new file for the exercise or make changes in the existing file.
Thank You.

changes_screenshot

**Review by SamuelBFavarin — APPROVED** [review]
LGTM 🚀

**Pull request merged**